src/main/webapp/index.html

```diff
@@ -1,46 +1,34 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <meta charset="utf-8"/>
+  <head>
+    <meta charset="utf-8" />
     <meta
-            content="width=device-width, initial-scale=1, shrink-to-fit=no"
-            name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+      name="viewport"
     />
-    <meta content="STEP Capstone: Techish" name="description"/>
-    <meta content="Samuel Sandoval and Hadley Shapland" name="author"/>
+    <meta content="STEP Capstone: Techish" name="description" />
```

Comment:
The style suggests not closing void elements.

https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity